cocotb/share/include/vpi_user.h

```diff
@@ -853,6 +853,8 @@ typedef struct t_cb_data
 #define cbNBASynch               30
 #define cbAtEndOfSimTime         31
 
+/**************************** Verilog AMS ****************************/
+#define vpiRealNet             526
```

Comment:
Actually, I had added this constant already to ``vpi_user_ext.h``, so this part of the PR is not necessary.